pkg/util/azureclient/mgmt/insights/activitylogs.go

```diff
@@ -4,25 +4,38 @@ package insights
 // Licensed under the Apache License 2.0.
 
 import (
-	mgmtinsights "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure"
+
+	//"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
```

Comment:
I would just remove these comments in `import` block. I think it is clear what to do from your comments

Reply:
Is this a requirement? Or a suggestion?

Reply:
I see no reason to keep this line: it does not add any value.

Reply:
Removed.